src/libstd/Cargo.toml

```diff
@@ -23,6 +23,9 @@ compiler_builtins = { path = "../libcompiler_builtins" }
 std_unicode = { path = "../libstd_unicode" }
 unwind = { path = "../libunwind" }
 
+[target.x86_64-unknown-linux-gnu.dependencies]
```

Comment:
["LeakSanitizer is only supported on x86_64 Linux."](http://clang.llvm.org/docs/LeakSanitizer.html)